code/obj/machinery/shield_generators/shield_generator.dm

```diff
@@ -133,7 +133,7 @@
 
 		if(diff > 1500)
 			lastuse = world.timeofday
-			user.show_text("You flip the switch on [src].")
+			visible_message("You flip the switch on [src].","You feel around the object and accidentally hit a switch")
```

Comment:
This is phrased a user.visible_message output, which you probably intended to do.